tests/Fixer/Basic/NoTrailingCommaInSinglelineFixerTest.php

```diff
@@ -0,0 +1,428 @@
+            '<?php if ($a) { echo 1;} foo($a,);',
+        ];
+
+        yield 'test method call' => [
+            '<?php $o->abc($a);',
+            '<?php $o->abc($a,);',
+        ];
+
+        yield 'nested call' => [
+            '<?php $o->abc($a,foo(1));',
+            '<?php $o->abc($a,foo(1,));',
+        ];
+
+        yield 'wrapped' => [
+            '<?php echo (new Process())->getOutput(1);',
+            '<?php echo (new Process())->getOutput(1,);',
+        ];
+
+        yield 'dynamic function and method calls' => [
+            '<?php $b->$a(1); $c("");',
+            '<?php $b->$a(1,); $c("",);',
+        ];
+
+        yield 'static function call' => [
+            '<?php
+unset($foo->bar);
+$b = isset($foo->bar);
+',
+            '<?php
+unset($foo->bar,);
+$b = isset($foo->bar,);
+',
+        ];
+
+        yield 'unset' => [
+            '<?php A::foo(1);',
+            '<?php A::foo(1,);',
+        ];
+
+        yield 'anonymous_class construction' => [
+            '<?php new class(1, 2) {};',
+            '<?php new class(1, 2,) {};',
+        ];
+
+        yield 'array/property access call' => [
+            '<?php
+$a = [
+    "e" => static function(int $a): void{ echo $a;},
+    "d" => [
+        [2 => static function(int $a): void{ echo $a;}]
+    ]
+];
+
+$a["e"](1);
+$a["d"][0][2](1);
+
+$z = new class { public static function b(int $a): void {echo $a; }};
+$z::b(1);
+
+${$e}(1);
+$$e(2);
+$f(0)(1);
+$g["e"](1); // foo',
+            '<?php
+$a = [
+    "e" => static function(int $a): void{ echo $a;},
+    "d" => [
+        [2 => static function(int $a): void{ echo $a;}]
+    ]
+];
+
+$a["e"](1,);
+$a["d"][0][2](1,);
+
+$z = new class { public static function b(int $a): void {echo $a; }};
+$z::b(1,);
+
+${$e}(1,);
+$$e(2,);
+$f(0,)(1,);
+$g["e"](1,); // foo',
+        ];
+
+        yield 'do not fix' => [
+            '<?php
+                function someFunction ($p1){}
+                function & foo($a,$b): array { return []; }
+
+                foo (
+                    1,
+                    2,
+                );
+
+                $a = new class (
+                    $a,
+                ) {};
+
+                isset($a, $b);
+                unset($a,$b);
+                list($a,$b) = $a;
+
+                $a = [1,2,3,];
+                $a = array(1,2,3,);
+
+                function foo1(string $param = null ): void
+                {
+                }
+            ;',
+        ];
+    }
+
+    /**
+     * @dataProvider provideFix80NoTrailingCommaInSinglelineFunctionCallFixerCases
+     * @requires PHP 8.0
+     */
+    public function testFix80NoTrailingCommaInSinglelineFunctionCallFixer(string $expected, string $input = null): void
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public static function provideFix80NoTrailingCommaInSinglelineFunctionCallFixerCases(): iterable
+    {
+        yield [
+            '<?php function foo(
+    #[MyAttr(1, 2,)] Type $myParam,
```

Comment:
Why not removing the trailing comma in the attribute?

Reply:
This is a test that is not touched:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.10.0/tests/Fixer/FunctionNotation/NoTrailingCommaInSinglelineFunctionCallFixerTest.php#L214

Reply:
Would be nice to add support for it, but it is not the goal of the PR and it is already pretty big.